Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7
+
+RUN pip install snowchange==0.0.2
```

Comment:
Can the pip version number match our current version number?

Reply:
I removed versioning inside Dockerfile, since in the README.md, the example include snowchange package installation with --upgrade flag already